proposed/http-client/http-client-meta.md

```diff
@@ -41,8 +40,7 @@ specify the default behaviours.
 The intention of this PSR is ensure library developers that all HTTP clients have the same 
 **default behavior**. That means that all HTTP clients MUST follow Liskov substitution principle
 when no configuration is provided. The PSR does not try to restrict nor define configuration for 
-HTTP clients. An implementing library is free to be configured by the application author to follow
-redirects, to throw exceptions or any other possible setting.  
+HTTP clients.
```

Comment:
This might be an issue. Consumers can't know whether redirects are followed or not and might fail if the client is configured to not follow any redirects.